nodejs/week2/homework/warmup/index.js

```diff
@@ -0,0 +1,30 @@
+const express = require("express");
+const app = express();
+
+app.get("/", (req, res) => res.send("nodejs week2 homework"));
+
+app.get("/numbers/add", async (request, response) => {
+  if (isNaN(request.query.first) || isNaN(request.query.second)) {
+    response.status(406).json(`Error! Not a number. Please enter 2 numbers`);
```

Comment:
I assume you know the difference between using response.json and response.send as you have used them both several times in your code.
For your reference. https://medium.com/gist-for-js/use-of-res-json-vs-res-send-vs-res-end-in-express-b50688c0cddf